pluralize.js

```diff
@@ -179,6 +179,21 @@
     return (inclusive ? count + ' ' : '') + pluralized;
   }
 
+  /**
+   * Make a word posessive handling plurals properly.
+   *
+   * @type {Function}
+   */
+  pluralize.possessive = function (word) {
+    var possessiveWord = word + "'s";
+
+    if (word.endsWith('s') && pluralize.isPlural(word)) {
```

Comment:
Also note that `endsWith` limits the JavaScript engines this can run on and changes compatibly in a breaking way of this package.

Reply:
I had no idea. Thanks :)